HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/InvoicesList/InvoicesListViewModel.swift

```diff
@@ -62,17 +62,20 @@ final class InvoicesListViewModel {
     let dispatchGroup = DispatchGroup()
     var shouldRefetchExtractions = false
     var documentIdToRefetch: String?
+    let shouldUseAlternativeNavigation: Bool
 
     init(coordinator: InvoicesListCoordinator,
          invoices: [DocumentWithExtractions]? = nil,
          documentService: GiniHealthSDK.DefaultDocumentService,
          hardcodedInvoicesController: HardcodedInvoicesControllerProtocol,
-         health: GiniHealth) {
+         health: GiniHealth,
+         shouldUseAlternativeNavigation: Bool) {
```

Comment:
[Clarification]: wha exactly do we mean here when we have `shouldUseAlternativeNavigation`?
Can we have a default value?